e2e/dashboard/dashboard.feature

```diff
@@ -70,9 +70,11 @@ Feature: Test to verify Dashboard links
     And she selects "Mention" integration action
     Then she is prompted to select a "Finish" connection from a list of available connections
 
-    When Camilla selects the "Salesforce Example" connection
-    And she selects "Create Opportunity" integration action
-    And Camilla clicks on the "Done" button
+    When Camilla selects the "HTTP Example" connection
```

Comment:
This is a good example, but I guess with should make it a bit better and specify correct properties in syndesis-rest jsondb. Because single URI is not valid anymore.